pkgs/development/libraries/raylib/default.nix

```diff
@@ -4,9 +4,15 @@
   alsaSupport ? stdenv.hostPlatform.isLinux, alsa-lib,
   pulseSupport ? stdenv.hostPlatform.isLinux, libpulseaudio,
   sharedLib ? true,
-  includeEverything ? true
+  includeEverything ? true,
+  waylandSupport
 }:
 
+let
+  glfw-wayland = glfw.override {
+    waylandSupport = true;
+  };
+in
```

Comment:
How about, instead of this let statement, down in `buildInputs` we use something like this:

```nix
buildInputs = [
# ...
(glfw.override { inherit waylandSupport; })
# ...
]
```

?

Reply:
Thanks, this seems to be exactly what I was looking for.